core/src/main/resources/jenkins/management/Messages.properties

```diff
@@ -22,17 +22,17 @@
 # THE SOFTWARE.
 #
 
-ConfigureLink.DisplayName=Configure System
+ConfigureLink.DisplayName=System
```

Comment:
Please remove all translations that are no longer appropriate or re-translate them, e.g. https://github.com/janfaracik/jenkins/blob/d52812a8f9e86465195f3d0d8f1ba6056f01f27a/core/src/main/resources/jenkins/management/Messages_bg.properties#L23 and many others